docs/activities_ActivityUsageEntity.js.html

```diff
@@ -36,6 +36,8 @@ <h1 class="page-title">Source: activities/ActivityUsageEntity.js</h1>
 /**
  * ActivityUsageEntity class representation of a d2l activity usage.
  */
```

Comment:
this file is generated noise, yeah?

Reply:
It's generated yeah, from the comments for the JSdocs